lib/api/CInferenceModelDefinition.cc

```diff
@@ -368,12 +364,8 @@ void CTrainedModel::classificationWeights(const TDoubleVec& classificationWeight
     m_ClassificationWeights = classificationWeights;
 }
 
-void CInferenceModelDefinition::fieldNames(TStringVec&& fieldNames,
-                                           std::size_t dependentVariableColumnIndex) {
-    m_FieldNames = fieldNames;
-    fieldNames.erase(fieldNames.begin() +
-                     static_cast<std::ptrdiff_t>(dependentVariableColumnIndex));
```

Comment:
Question: does it not matter we include control field names here now, i.e. fields `.`?

Reply:
It does not, since we only return the field names actually used by the models.

Reply:
This is a nice simplification in that case!